content/guides/intro/token-explained.md

```diff
@@ -0,0 +1,190 @@
+- **Asking the blockchain:** The RPC sends the translated question to the Solana
+  blockchain.
+- **Blockchain's answer:** The blockchain finds your balance and sends the
+  answer back to the RPC.
+- **Translating back:** The RPC translates the answer back into English.
+- **Website gets the info:** The RPC sends the translated answer to the website.
+- **You get your answer:** The website shows you your balance in a way you
+  understand!
+
+**Key point:** The Solana RPC makes everything work smoothly, so you don't have
+to worry about learning the complicated language of the blockchain.
+
+## Exploring Different Types of Accounts in Solana
+
+![image](https://github.com/Ayushjhax/developer-content/assets/116433617/cf253051-a943-43d1-a56e-37e8adc39f46)
+
+Now, let's dive into the different types of accounts in Solana, each playing a
+crucial role in the ecosystem.
+
+### Wallet Account
+
+This is like your main wallet where you stash your SOL coins (Solana's own
+money). It’s used to derive the accounts for your tokens.
+
+### Token Account
+
+These accounts are associated with users or wallets and hold vital information
+about the tokens. They work in coherence with the wallet account and mint
+account, sharing similar data and intent for performing transactions.
+
+### Mint Account
+
+The mint account stores data about the token itself, including properties such
+as mint authority, supply, decimals, and freeze authority. Think of these as the
+instruction manuals for each token. They tell you how many tokens are out there,
+who's allowed to make more, that sort of thing.
+
+![image](https://github.com/Ayushjhax/developer-content/assets/116433617/e325ff8a-c020-4ca6-91ad-dda7ea83a32b)
+
+- **Mint authority:** The designation of your own or some other program using
+  which the mint tokens can be signed.
+- **Supply:** Holds information on how many tokens have been issued overall.
+- **Decimals:** The limit until which the token can be broken down into.
+- **Is Initialized:** Flag to see if the account is ready to go (or not).
+- **Freeze authority:** The designation of your own or some other program using
+  which the mint tokens can be frozen.
+
+### Metadata Account
+
+These accounts hold metadata of mint accounts, storing information about the
+token, such as its name, symbol, description, and image. These are like fancy
+labels on your token boxes. They've got the token's name, its ticker symbol
+(like RAD for RadCoin), maybe even a cool picture.
+
+**Example:** Currency has a name (US Dollar), symbol ($ or USD), description,
+and image. Solana’s metadata holds another layer of metadata:
+
+- NAME
+- SYMBOL
+- DESCRIPTION
+- IMAGE
+
+## Deep Dive into Tokens
+
+### Creation and Functionality
+
+Tokens in Solana are created using the Token Program, which acts as the Token
+Factory. This program allows users to mint, transfer, and burn tokens between
+accounts, functioning as a factory of accounts that enable token
+functionalities.
```

Comment:
```suggestion
Tokens in Solana are created using the Token Program. This program allows users to create token mints, mint tokens, transfer tokens, and burn tokens.
```

Reply:
I've removed the 'factory' analogy as we use this elsewhere (not just in this document) for token mint accounts.